.github/workflows/generate_bindings.yaml

```diff
@@ -0,0 +1,111 @@
+name: Generate bindings
+
+on:
+  workflow_dispatch:
+
+jobs:
+  generate-bindigs:
+    name: Generate bindings
+    strategy:
+      matrix:
+        include:
```

Comment:
Doesn't this list now miss `aarch64-pc-windows-msvc`?

Reply:
Forgot, fixed now